pkg/types/ccipocr3/interfaces.go

```diff
@@ -53,3 +53,9 @@ type ExecutePluginCodec interface {
 type MessageHasher interface {
 	Hash(context.Context, CCIPMsg) (Bytes32, error)
 }
+
+type HomeChainPoller interface {
+	GetConfig() HomeChainConfig
```

Comment:
This needs to be either scoped to a particular chain selector or we get all the chain configs in one call. I think having both is more flexible at this stage.

```suggestion
	GetChainConfig(ctx context.Context, chainSelector uint64) HomeChainConfig
	GetAllChainConfigs(ctx context.Context) map[uint64]HomeChainConfig
```